deployments/regenFarms/config.ts

```diff
@@ -4,19 +4,19 @@ import { IRegenConfig } from "../types";
 const config: IRegenConfig = {
     alreadyDeployedTokenDistroAddress: "",
     newTokenDistroParams: {
-        startTime: 1648857600, // Sunday, 20 February 2022 10:00:00
+        startTime: 1649001600, // Sunday, 03 April 2022 16:00:00 GMT
         cliffPeriod: 0,
-        duration: 157_680_000, // 5 years * 365 days * 24 hours * 3600 seconds = 157680000
+        duration: 149_040_000, // 5 years * 365 days * 24 hours * 3600 seconds = 157680000
         initialPercentage: 10_00, // two decimals of precision
-        tokenAddress: "0x6c16216484069C19530a57762AD6630fB678D00E", // TestFOX
-        totalTokens: "1000000000", // 1 Billion
+        tokenAddress: "0x21a42669643f45bc0e086b8fc2ed70c23d67509d", // FOX
+        totalTokens: "1000001337", // 1 Billion
         cancelable: true,
     },
     unipools: {
-        TestFOX_TestHNY: {
-            uniTokenAddress: "0x6a5689435Fb27f235f4cfe05ADDabf8fF173f0Ea", // HoneySwap DRGIV3/TestELK
+        FOX_HNY: {
+            uniTokenAddress: "0x8a0bee989c591142414ad67fb604539d917889df", // HoneySwap DRGIV3/TestELK
             lmDuration: 1_209_600, // 2 weeks * 7 days * 24 hours * 3600 seconds = 1_209_600
-            rewardAmount: "100000", // The value to assign on tokendistro
+            rewardAmount: "327387.1731240768", // The value to assign on tokendistro
```

Comment:
Where'd this number come from?

Reply:
Found: `327387.1731240768 == 334,068.54400416 * 0.98`. From @willyogo at https://discord.com/channels/@me/948593968197812255/958816862886559794:
```
From this other gnosis safe (https://gnosis-safe.io/app/gno:0x25A6343E69CE6cd7a2c34536595341Cb6b038c5F/balances) which is a joint multisig with 1hive, we will transfer 334,068.54400416 FOX.
1% to giveth liquidity multisig (0xf924fF0f192f0c7c073161e0d62CE7635114e74f)
1% to donation.eth
98% to the token distro contract. can you please provide the token distro address @Amin ?
```